### PR TITLE
docs: Update README to include Jenkins and github actions documentation

### LINKED ISSUE #3565: update README to include Jenkins and github actions documentation

### PR DESCRIPTION
**Pull request by GabyCT**
This PR updates the README to start including more information about
Jenkins jobs and github actions that are being used in kata repositories.

Fixes #3565

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +78,20 @@ Use `make list-install-targets` to retrieve all the available install targets.
 > CI. Some of this setup runs as the `root` user and **could break your developer's
 > system**. See [Developer Mode](#developer-mode).
 
+### Controlling the CI
+
+#### Github Actions
+
+Kata uses Github Actions in the [kata-containers](https://github.com/kata-containers/kata-containers) repos.
+There are no trigger phrases to invoke these actions as they are triggered by creating a pull request.
```
Comment (fidencio) [inline README.md:86]: We actually have one trigger phrase used with GitHub actions: `/test_kata_deploy`

File: README.md [hunk 2]
```diff
@@ -77,6 +78,20 @@ Use `make list-install-targets` to retrieve all the available install targets.
...
+
+#### Jenkins
+
+The Jenkins configuration and most documentation is kept in the [CI repository](https://github.com/kata-containers/ci).
```
Comment (fidencio) [inline README.md:90]: Not related to this PR, but this reminds me that we need urgently to update the CI page and remove everything that's 1.x specific. :-)

File: README.md [hunk 3]
```diff
@@ -77,6 +78,20 @@ Use `make list-install-targets` to retrieve all the available install targets.
...
+Jenkins is setup to trigger a CI run on all the slaves/nodes when a "/test" comment is added to a pull request. However,
+there are some specific comments that are defined for specific CI slaves/nodes which are defined in the Jenkins
+`config.xml` files in the `<triggerPhase>` XML element in the [CI repository](https://github.com/kata-containers/ci).
```
Comment (fidencio) [inline README.md:93]: What about also adding a link to https://github.com/kata-containers/community/wiki/Controlling-the-CI ?

**Review by fidencio — COMMENTED** [review]
@GabyCT, I have added a few suggestions.

File: README.md (hunk 3 above)
Comment (GabyCT) [reply README.md:92]: @fidencio for this PR I do not want to include a link to that wiki as it is not updated and has a lot of information that is working with kata 2.x, I will create another PR after I update the wiki and put the current trigger phrases for kata 2.x ci jobs

Comment (GabyCT) [reply README.md:92]: @fidencio for this PR I do not want to include a link to that wiki as it is not updated and has a lot of information that is working with kata 2.x, I will create another PR after I update the wiki and put the current trigger phrases for kata 2.x ci jobs

Comment (fidencio) [reply README.md:93]: Okay, makes sense.

File: README.md [hunk 4]
```diff
@@ -77,6 +78,20 @@ Use `make list-install-targets` to retrieve all the available install targets.
 > CI. Some of this setup runs as the `root` user and **could break your developer's
 > system**. See [Developer Mode](#developer-mode).
 
+### Controlling the CI
+
+#### GitHub Actions
+
+Kata Containers uses GitHub Actions in the [Kata Containers](https://github.com/kata-containers/kata-containers) repos.
+The trigger phrase to invoke these actions is `/test_kata_deploy`.
```
Comment (fidencio) [inline README.md:86]:
```suggestion
All those actions, apart from the one to test kata-deploy, are automatically triggered when a pull-request is submitted.  For testing kata-deploy, one can use `/test_kata_deploy`.
```

**Review by fidencio — APPROVED** [review]
lgtm, thanks @GabyCT!

**Review by devimc — APPROVED** [review]

**Pull request merged**